examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-themes/core-components-clean/styles/components/accordion/base.less

```diff
@@ -40,7 +40,6 @@
         color: inherit;
         font: inherit;
         cursor: pointer;
-        outline: inherit;
```

Comment:
Shoud be `outline: inherit;` deleted from here too?
https://github.com/adobe/aem-core-wcm-components/blob/c6756447df4ebcfd66d133385502cae3135f87be/examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-themes/core-components-clean/styles/components/accordion/base.less#L43